Pyrado/pyrado/algorithms/utils.py

```diff
@@ -269,3 +269,11 @@ def reset_rollouts(self) -> None:
     def __getattr__(self, name: str) -> Any:
         # Forward to the wrapped sampler
         return getattr(self.wrapped_sampler, name)
+
+    def __getstate__(self):
+        """Required so that pickle does not end in an endless recursion"""
+        return vars(self)
+
+    def __setstate__(self, state):
+        """Required so that pickle does not end in an endless recursion"""
+        vars(self).update(state)
```

Comment:
is this the same as `self.__dict__["foo"] = bar`?

Reply:
More like `self.__dict__.update(state)` but if state is defined as follows its equivalent:
```python
state = {'foo' : bar }
```

Reply:
ok, thanks